### DISCUSSION TITLE
GraphQL in offline environments

### DISCUSSION THREAD

**Discussion by science4reason** [strapi/strapi#7389] [Legacy Topics]
We run Strapi 3.1.1 in an offline environment(behind a corporate  FW without Internet connection). GraphQL playground throws some erros to the Developer Console and Network tab. Seems Strapi preloads some stuff from a CDN resource. This causes the loading of the playground to fail. I see this was an issue with a native Apollo playground. 

Anyone found a workaround/config for this?

graphql:7 GET https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700|Source+Code+Pro:400,700 net::ERR_INTERNET_DISCONNECTED
graphql:13 GET https://cdn.jsdelivr.net/npm/@apollographql/graphql-playground-react@1.7.32/build/static/css/index.css net::ERR_INTERNET_DISCONNECTED
graphql:18 GET https://cdn.jsdelivr.net/npm/@apollographql/graphql-playground-react@1.7.32/build/static/js/middleware.js net::ERR_INTERNET_DISCONNECTED
graphql:548 could not find config
(anonymous) @ graphql:548
load (async)
(anonymous) @ graphql:532
cdn.jsdelivr.net/npm/@apollographql/graphql-playground-react@1.7.32/build/favicon.png:1 GET https://cdn.jsdelivr.net/npm/@apollographql/graphql-playground-react@1.7.32/build/favicon.png net::ERR_INTERNET_DISCONNECTED

**Comment by derrickmehaffy** [comment]
@alexandrebodin would is just be better to roll these assets into the plugin package instead of calling remote resources for them?

**Reply by alexandrebodin** [reply]
We cannot cahnge this as it is coming from an external lib right now :)

**Reply by derrickmehaffy** [reply]
Ahhh yeah good point, after looking at the code I see it. @science4reason this is something you will likely need to take up with https://github.com/prisma-labs/graphql-playground

**Reply by science4reason** [reply]
Thanks, will do so
